README.md

```diff
@@ -1,16 +1,19 @@
 # Project
 
-This repository is a fork of the original YCSB repository. We had to create this fork as there was no response to our pull requests from the YCSB maintainers.
+This repository is a fork of the original YCSB repository in order to support Azure workloads. We're working with the 
+upstream YCSB team to get these changes merged into the main repository.
```

Comment:
With all these changes, I think it's time to reevaluate the strategy of going back to the original YCSB where there is no PR merge in last 2 years. Also, how can we get developers community to use and contribute this open source?

Reply:
My bad on this, I did get some contact with the YCSB folks but didn't chase them hard enough in recent months.  I'll reach out again and see if that helps.

Reply:
I'll land this PR and then head back upstream